src/main/java/no/nav/pto/veilarbportefolje/arbeidssoeker/v2/ArbeidssoekerDataRepository.kt

```diff
@@ -0,0 +1,91 @@
+package no.nav.pto.veilarbportefolje.arbeidssoeker.v2
+
+import no.nav.common.types.identer.Fnr
+import no.nav.pto.veilarbportefolje.database.PostgresTable.OPPLYSNINGER_OM_ARBEIDSSOEKER
+import no.nav.pto.veilarbportefolje.database.PostgresTable.OPPLYSNINGER_OM_ARBEIDSSOEKER_JOBBSITUASJON
+import no.nav.pto.veilarbportefolje.database.PostgresTable.SISTE_ARBEIDSSOEKER_PERIODE
+import no.nav.pto.veilarbportefolje.database.PostgresTable.PROFILERING
+import no.nav.pto.veilarbportefolje.util.DateUtils
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.stereotype.Repository
+
+@Repository
+class ArbeidssoekerDataRepository(
+    private val db: JdbcTemplate
```

Comment:
we have two jdbctemplates...which one you are choosing here?

Reply:
import org.springframework.jdbc.core.JdbcTemplate

which should i use? ;o

Reply:
We have one that is readOnly and other that can be used to write data..